client/blocks/ecommerce-manage-nudge/index.js

```diff
@@ -0,0 +1,101 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import PropTypes from 'prop-types';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import isECommerceManageNudgeDismissed from './selectors';
+import QueryPreferences from 'components/data/query-preferences';
+import { dismissNudge } from './actions';
+import { enhanceWithSiteType, recordTracksEvent } from 'state/analytics/actions';
+import { withEnhancers } from 'state/utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ECommerceManageNudge extends Component {
+	static propTypes = {
+		isDismissed: PropTypes.bool.isRequired,
+		recordTracksEvent: PropTypes.func.isRequired,
+		siteId: PropTypes.number.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
+	componentDidMount() {
+		this.recordView();
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.siteId && this.props.siteId && this.props.siteId !== prevProps.siteId ) {
+			this.recordView();
+		}
+	}
+
+	recordView() {
+		if ( ! this.props.isDismissed ) {
+			this.props.recordTracksEvent( 'calypso_ecommerce_manage_stats_nudge_view' );
+		}
+	}
+
+	onDismissClick = () => {
+		this.props.recordTracksEvent( 'calypso_ecommerce_manage_stats_nudge_dismiss_icon_click' );
+		this.props.dismissNudge();
+	};
+
+	render() {
+		const { translate } = this.props;
+
+		if ( this.props.isDismissed ) {
+			return null;
+		}
+
+		return (
+			<Card className="ecommerce-manage-nudge">
+				<QueryPreferences />
+				<Gridicon
+					icon="cross"
+					className="ecommerce-manage-nudge__close-icon"
+					onClick={ this.onDismissClick }
+				/>
+				<div className="ecommerce-manage-nudge__body">
+					<div className="ecommerce-manage-nudge__image-wrapper">
+						<img
+							className="ecommerce-manage-nudge__image"
+							src="/calypso/images/extensions/woocommerce/woocommerce-setup.svg"
+							alt=""
+						/>
+					</div>
+					<div className="ecommerce-manage-nudge__info">
+						<h3 class="ecommerce-manage-nudge__title">Start managing your Store.</h3>
```

Comment:
Needs a `translate()`

Reply:
And I'm getting an exception on this line for the use of `class` vs `className`

![image](https://user-images.githubusercontent.com/22080/52592180-90726200-2dfa-11e9-9d29-ee2b9b7fca19.png)

Reply:
Thanks for catching that... fixed in https://github.com/Automattic/wp-calypso/pull/30548/commits/ca47a2d676364416c3319d81bf1944d532252d34.

I'll get this merged tomorrow.